driver_scripts/setup_ocean_region_groups.py

```diff
@@ -91,7 +90,7 @@ def remove_small_polygons(inFileName, outFileName, minArea):  # {{{
     write_all_features(features, outFileName, indent=4)  # }}}
 
 
-def build_basins():  # {{{
+def build_ocean_basins():  # {{{
     basinGroupName = 'OceanBasinRegionsGroup'
     basinFileName = 'oceanBasins.geojson'
```

Comment:
@milenaveneziani, should this feature just be -180 to 180 lon, -90 to 90 lat or should it be the union of all ocean basins without land?